src/App.tsx

```diff
@@ -1,12 +1,63 @@
 import React from 'react';
 import './App.scss';
 
-// import { getAll, get5First, getRed } from './api/goods';
+import { GoodsList } from './components/GoodsList';
+import { getAll, get5First, getRed } from './api/goods';
 // or
 // import * as goodsAPI from './api/goods';
 
-const App: React.FC = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+interface State {
+  goods: Good[];
+}
+
+class App extends React.Component<{}, State> {
+  state = {
+    goods: [],
+  };
+
+  handleClick = async (callback: () => Promise<Good[]>) => {
```

Comment:
Main goal of this function is to fetch goods.
```suggestion
  getGoods = async (callback: () => Promise<Good[]>) => {
```